packages/coingecko/package.json

```diff
@@ -36,13 +36,12 @@
     "react": "^17.0.1"
   },
   "dependencies": {
-    "@ethersproject/contracts": "5.6.0",
-    "@ethersproject/providers": "5.6.2",
     "@usedapp/core": "workspace:*",
     "isomorphic-fetch": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "*"
+    "react": "*",
+    "ethers": "*"
```

Comment:
Don't we need ethers in dev dependencies? Or maybe we don't need ethers here at all?

Reply:
This one is needed in case someone installs `@usedapp/coingecko` and not `@usedapp/core`. Because peer dependencies are not transitive, we need to include `ethers` in peer dependencies here as well.